about.css

```diff
@@ -256,4 +272,66 @@ button:disabled {
   line-height: 52px;
   letter-spacing: 0;
   text-align: left;
+  width: 315px;
+}
+
+@media only screen and (min-width: 768px) {
+  .projects-heading {
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .projects-heading > div {
+    width: 60%;
+    display: flex;
+    align-items: center;
+  }
+
+  .project1-image {
+    padding-right: 24px;
+  }
+
+  .project1-card {
+    flex-direction: row;
+    align-items: start;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] Great job so far :clap: but as I mentioned in my previous review whenever hover over any card the `overlay` will disappear and pic will be shown clearly check below :point_down: the Figma and your design comparison :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-03-30 14-15-27](https://user-images.githubusercontent.com/45226848/228791530-75427e2b-1a42-4298-908e-4f28458730f2.png)

**Figma Design**
![Screenshot from 2023-03-29 23-58-57](https://user-images.githubusercontent.com/45226848/228791424-86652267-5539-42a0-85d4-f6587dc33cf0.png)

- [ ] If you check your design the picture is not shown clearly but if you check Figma design the Picture is clear whenever someone is hovering over any card so kindly fix this issue also :+1: